deploy/resource-quotas/Makefile

```diff
@@ -0,0 +1,28 @@
+SHELL = /usr/bin/env bash -e
```

Comment:
Why not incorporate this into the main `Makefile`?

Reply:
The Makefile in the root directory does not call this one. The intended use is for people to run `make clean all` and commit the changes.

Reply:
The main in / is also generating resources to be committed.  It's reasonable for this to be incorporated in a single Makefile.

Reply:
See https://github.com/openshift/managed-cluster-config/pull/23/commits/84f1b4a3e60de78c8f07fc455c2f32fd41fd10db